influxdb-backup/README.md

```diff
@@ -0,0 +1,173 @@
+# BUILD SETUP
+
+```sh 
+
+cmurugan@iotserver:~/iot/docker-ttn-dashboard$ docker-compose up -d
+Creating network "dockerttndashboard_default" with the default driver
+Creating dockerttndashboard_influxdb_1 ...
+Creating dockerttndashboard_influxdb_1 ... done
+Creating dockerttndashboard_influxdb-backup_1 ...
+Creating dockerttndashboard_node-red_1 ...
+Creating dockerttndashboard_grafana_1 ...
+Creating dockerttndashboard_grafana_1
+Creating dockerttndashboard_node-red_1
+Creating dockerttndashboard_grafana_1 ... done
+Creating dockerttndashboard_apache_1 ...
+Creating dockerttndashboard_apache_1 ... done
+
+```
+
+## status of docker container and databases
+
+```sh
+
+cmurugan@iotserver:~/iot/docker-ttn-dashboard$ docker-compose ps
+                Name                              Command               State                    Ports
+------------------------------------------------------------------------------------------------------------------------
+dockerttndashboard_apache_1            /bin/bash /root/setup.sh         Up      0.0.0.0:443->443/tcp, 0.0.0.0:80->80/tcp
+dockerttndashboard_apache_run_1        /bin/bash                        Up
+dockerttndashboard_grafana_1           /run.sh                          Up      3000/tcp
+dockerttndashboard_influxdb-backup_1   /entrypoint.sh influxd           Up      8086/tcp
+dockerttndashboard_influxdb_1          /entrypoint.sh influxd           Up      8086/tcp
+dockerttndashboard_node-red_1          npm start -- --userDir /da ...   Up      1880/tcp
+cmurugan@iotserver:~/iot/docker-ttn-dashboard$
+
+
+cmurugan@iotserver:~/iot/docker-ttn-dashboard$ docker-compose exec influxdb influx
+Connected to http://localhost:8086 version 1.2.4
+InfluxDB shell version: 1.2.4
+> show databases
+name: databases
+name
+----
+_internal
+
+> create database testdb
+> show databases
+name: databases
+name
+----
+_internal
+testdb
+
+> use testdb
+Using database testdb
+> INSERT cpu,host=serverA,region=us_west value=0.64
+> SELECT * FROM cpu
+name: cpu
+time                host    region  value
+----                ----    ------  -----
+1508942563099443156 serverA us_west 0.64
+
+```
+
+# BACKUP DATABASE THROUGH SHELL SCRIPT USING EXTRA INFLUXDB INSTANCE
```

Comment:
This should be `##`.  I suggest using Visual Studio Code and their tools for Markdown maintenance, as they have a GFM linter, very useful.